src/pages/patientView/mutation/column/FACETSClonalColumnFormatter.tsx

```diff
@@ -0,0 +1,213 @@
+                </span>
+               );
+    }
+
+    public static getTooltip(sampleId:string, clonalValue:string, ccfMCopies:string, sampleManager:SampleManager) {
+        let clonalColor = "";
+        if (clonalValue === "yes") {
+            clonalColor = "limegreen";
+        } else if  (clonalValue === "no") {
+            clonalColor = "dimgrey";
+        } else {
+            clonalColor = "lightgrey";
+        }
+        return (
+                <div>
+                        <table>
+                                <tr><td style={{paddingRight:10}}>{sampleManager.getComponentForSample(sampleId, 1, "")}</td><td><strong></strong></td></tr>
+                                <tr><td style={{paddingRight:5}}>Clonal</td><td><span style={{color: `${clonalColor}`, fontWeight: "bold"}}>{clonalValue}</span></td></tr>
+                                <tr><td style={{paddingRight:5}}>CCF</td><td><strong>{ccfMCopies}</strong></td></tr>
+                        </table>
+                </div>
+        );
+    }
+
+    public static getClonalCircle(clonalValue:string) {
+        let color:string = "";
+        if (clonalValue === "yes") {
+            color = "limegreen";
+        } else if (clonalValue === "no") {
+            color = "dimgrey";
+        } else {
+            color = "lightgrey";
+        }
+        return (
+                <svg height="10" width="10">
+                    <circle cx={5} cy={5} r={5} fill={`${color}`}/>
+                </svg>
+        );
+    }
+
+    public static getCcfMCopiesUpperValue(mutations:Mutation[]):number {
+        const ccfMCopiesUpperValue = mutations[0].ccfMCopiesUpper;
+        return ccfMCopiesUpperValue;
+    }
+
+    public static getCcfMCopiesValue(mutations:Mutation[]):number {
+        const ccfMCopiesValue = mutations[0].ccfMCopies;
+        return ccfMCopiesValue;
+    }
+
+    public static getClonalValue(mutations:Mutation[]):string {
+        let textValue:string = "";
+        const ccfMCopiesUpperValue = FACETSClonalColumnFormatter.getCcfMCopiesUpperValue(mutations);
+        if (floatValueIsNA(ccfMCopiesUpperValue)) {
+            textValue = "";
+        } else if (ccfMCopiesUpperValue === 1) {
+            textValue = "yes";
+        } else {
+            textValue = "no";
+        }
+        return textValue;
+    }
+
+    public static renderFunction(mutations:Mutation[], sampleIds:string[], sampleManager:SampleManager|null) {
+        if (!sampleManager) {
+            return (<span></span>);
+        }
+        return FACETSClonalColumnFormatter.getDisplayValue(mutations, sampleIds, sampleManager);
+    }
+
+    public static getClonalDownload(mutations:Mutation[]): string|string[] {
+        let result = [];
+        if (mutations) {
+            for (let mutation of mutations) {
+                result.push(FACETSClonalColumnFormatter.getClonalValue([mutation]));
+            }
+        }
```

Comment:
it looks as if calls to `getClonalDownload()` all expect an array. I think we can remove the check for whether the length of the result is just 1 and make this always return an array.